managed/ui/src/components/tables/CreateBackup/CreateBackup.js

```diff
@@ -81,21 +81,34 @@ export default class CreateBackup extends Component {
 
     if (isDefinedNotNull(values.storageConfigUUID)) {
       let backupType = null;
+      let frequency = null;
+
       if (this.state.backupType === 'ysql') {
         backupType = YSQL_TABLE_TYPE;
       } else if (this.state.backupType === 'ycql') {
         backupType = YCQL_TABLE_TYPE;
       } else if (this.state.backupType === 'yedis') {
         backupType = YEDIS_TABLE_TYPE;
       }
+
+      if (!isEmptyString(values.schedulingFrequency)) {
+        if (values?.schedulingFrequencyUnit?.value === 'Hours') {
+          frequency =
+            values?.schedulingFrequency % 1
+              ? (values?.schedulingFrequency % 1) * 3600 * 1000 +
+                Math.floor(values?.schedulingFrequency) * 3600 * 1000
+              : values?.schedulingFrequency * 3600 * 1000;
```

Comment:
why not just `frequency = values?.schedulingFrequency * 3600 * 1000`? it seems the same math here
and I'd suggest ensuring the `values` is defined rather than put `?` everywhere